python/API_Examples_Python.py

```diff
@@ -0,0 +1,200 @@
+                 'sig': signature}
+    post_data = json.dumps(post_data)
+    order_response = requests.post(mainnet_url + "/v1/placeorder", post_data, headers=headers)
+    if order_response.status_code == requests.codes.ok:
+        pprint.pprint(order_response.json())
+    else:
+        print(order_response.content)
+        print("ERROR: Place Order post did not work. ")
+
+def cancel_order():
+    """
+    cancel a single specific order
+    """
+    headers = {"Authorization": user_information["token"]}
+    pay_load = {"traderAddr": user_information["traderAddr"],
+                "orderId": "10827893",
+                "pairId": "ETH_ZIL",
+                "nonce": "2640819234038"}
+    pay_load = json.dumps(pay_load)
+    cancel_order_response = requests.post(mainnet_url + "/v1/cancelorder", pay_load, headers=headers)
+    if cancel_order_response.status_code == requests.codes.ok:
+        print("ok")
+    else:
+        print("ERROR: Cancel Order post did not work. ")
+
+def cancel_all_orders():
+    """
+    # Cancel all existing orders using given parameters
+    """
+    headers = {"Authorization": user_information["token"]}
+    pay_load = {"traderAddr": user_information["traderAddr"],
+                "pairId": 'ETH_ZIL',
+                "nonce": '3640819234038'}
+    pay_load = json.dumps(pay_load)
+    response = requests.post(mainnet_url + "/v1/cancelallorders", pay_load, headers=headers)
+    print(response.content)
+
+def get_active_orders():
+    """
+    Get unfilled or partially filled orders that have not been cancelled or expired of a trader.
+    """
+    headers = {"Authorization": user_information["token"]}
+    pay_load = {"pairId": 'ETH_ZIL',
+                "size": '100',
+                "page": '1',
+                "traderAddr": user_information["traderAddr"]}
+    url_params = "/v1/activeorders/" + pay_load["traderAddr"] + "/" + pay_load['pairId'] + "/" + pay_load['size'] + "/" + pay_load["page"]
+    pay_load = json.dumps(pay_load)
+    response = requests.get(mainnet_url + url_params, pay_load, headers=headers)
+    if response.status_code == requests.codes.ok:
+        pprint.pprint(response.json())
+    else:
+        print("ERROR: Get Active Orders request did not work. ")
+
+def get_trades():
+    """
+    Get recent trades on current wallet address.
+    """
+    headers = {"Authorization": user_information["token"]}
+    pay_load = {'pairId': 'ETH_ZIL',
+                'size': '100',
+                'traderAddr': user_information["traderAddr"]}
+    url_params = "/v1/trades/" + pay_load["traderAddr"] + "/" + pay_load["pairId"] + "/" + pay_load["size"]
+    pay_load = json.dumps(pay_load)
+    response = requests.get(mainnet_url + url_params, pay_load, headers=headers)
+    if response.status_code == requests.codes.ok:
+        pprint.pprint(response.json())
+    else:
+        print("ERROR: Get trades request did not work. ")
+
+
+if __name__ == '__main__':
+    user_login()
```

Comment:
This call should return a login token, which should be placed in `user_information["token"]`.